src/hooks/useRefreshAccountData.ts

```diff
@@ -37,7 +36,9 @@ export default function useRefreshAccountData() {
     queryClient.invalidateQueries(nftsQueryKey({ address: accountAddress, sortBy: nftSort }));
     queryClient.invalidateQueries(positionsQueryKey({ address: accountAddress as Address, currency: nativeCurrency }));
     queryClient.invalidateQueries(addysSummaryQueryKey({ addresses: allAddresses, currency: nativeCurrency }));
-    queryClient.invalidateQueries(userAssetsQueryKey({ address: accountAddress, currency: nativeCurrency, connectedToHardhat }));
+    queryClient.invalidateQueries(
+      userAssetsQueryKey({ address: accountAddress, currency: nativeCurrency, testnetMode: connectedToHardhat })
+    );
```

Comment:
again this can probably be removed if we're migrating off the old user assets

Reply:
i think we still need this otherwise user assets won't refresh upon pull to refresh